_data/SMR Files/EGAE/Ground Map

```diff
@@ -0,0 +1 @@
+
```

Comment:
Ground Map folder not required - generator will always prioritise SMR folder if there is one :)

Reply:
Was going by other examples of directories to store the associated .kmz file in for others...

Reply:
Oh I see now! Thought this was in the airport folder! My mistake 😄